packages/common/test/id.test.ts

```diff
@@ -41,3 +46,32 @@ test('idrequest', () => {
     expect(typeof output.ipArData[3].encPrfKeyShare).toEqual('string');
     expect(typeof output.ipArData[3].proofComEncEq).toEqual('string');
 });
+
+test('Create id recovery request', () => {
+    const ipInfo = JSON.parse(
+        fs.readFileSync('./test/resources/ip_info.json').toString()
+    ).value;
+    const globalContext = JSON.parse(
+        fs.readFileSync('./test/resources/global.json').toString()
+    ).value;
+    const seedAsHex =
+        'efa5e27326f8fa0902e647b52449bf335b7b605adc387015ec903f41d95080eb71361cbc7fb78721dcd4f3926a337340aa1406df83332c44c1cdcfe100603860';
+
+    const timestamp = Math.floor(Date.now() / 1000);
```

Comment:
Just a note that in general it can be dangerous to use dynamic dates in unit tests. The unit tests will run on different input on each run, so it is not consistently testing the same thing. It is generally better to hardcode it instead (if possible).

Reply:
Hardcoded it.